app/src/main/java/ca/cgagnier/wlednativeandroid/model/wledapi/Usermods.kt

```diff
@@ -0,0 +1,13 @@
+package ca.cgagnier.wlednativeandroid.model.wledapi
+
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import com.squareup.moshi.JsonQualifier
+
+
+@JsonClass(generateAdapter = true)
+data class Usermods (
+
+	@Json(name = "Battery level") val batLevel : List<Any>? = null,
```

Comment:
I've not tried the battery usermod, I find it strange that it would have a list of battery level. Could you share with me an example of the JSON in the "u" section for the battery please?

I think that since usermods can be very varied, maybe there should be a package `package ca.cgagnier.wlednativeandroid.model.wledapi.usermods` that contains a class for each supported usermods. Then this usermod class would do this instead:

```
// In ca.cgagnier.wlednativeandroid.model.wledapi, UserMods.kt
@Json(name = "Battery level") val batteryLevel : BatteryLevel? = null,
```
```
// in ca.cgagnier.wlednativeandroid.model.wledapi.usermods, BatteryLevel.kt
@JsonClass(generateAdapter = true)
data class BatteryLevel (
[...]
```

Reply:
``` json
"u": {
      "Battery level": [7, " %"],
      "Battery voltage": [4.27, " V"],
      "Next update": [0, " sec"],
      "AudioReactive": [
        "<button class=\"btn btn-xs\" onclick=\"requestJson({AudioReactive:{enabled:false}});\"><i class=\"icons on\">&#xe08f;</i></button>"
      ],
      "Audio Input Level": [
        "<div class=\"slider\"><div class=\"sliderwrap il\"><input class=\"noslide\" onchange=\"requestJson({AudioReactive:{inputLevel:parseInt(this.value)}});\" oninput=\"updateTrail(this);\" max=255 min=0 type=\"range\" value=128 /><div class=\"sliderdisplay\"></div></div></div>"
      ],
      "Audio Source": ["I2S digital", " - quiet"],
      "Sound Processing": ["running"],
      "Manual Gain": [1.56, "x"],
      "UDP Sound Sync": ["send mode", " v2"]
    },
```

This is the JSON of my usermods. As you can see there are no separated keys for each usermod.
BatteryLevel and voltage are an array because they include the unit of the number for whatever reason